packages/@aws-c2a/cdk-pipelines-step/package.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "@aws-c2a/cdk-pipelines-step",
+  "stability": "experimental",
+  "version": "0.3.3",
+  "description": "A CDK Pipelines Step that runs aws-c2a diff given a rule set",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "jsii": {
+    "outdir": "dist",
+    "targets": {
+      "java": {
+        "package": "software.amazon.awscdk.c2a.pipelinestep",
+        "maven": {
+          "groupId": "software.amazon.awscdk",
+          "artifactId": "c2a"
+        }
+      },
+      "python": {
+        "distName": "aws-cdk.c2a",
+        "module": "aws_cdk.c2a"
+      },
+      "dotnet": {
+        "namespace": "Amazon.CDK.AWS.C2A",
+        "packageId": "Amazon.CDK.AWS.C2A"
+      }
+    }
+  },
+  "scripts": {
+    "build": "yarn clean && yarn compile",
+    "build:watch": "yarn clean && npx jsii --watch",
+    "package": "jsii-pacmak",
```

Comment:
do i need to run package as part of my npm script to publish?

Reply:
Only if you're also going to publish to PyPI, Maven Central and NuGet. It's probably okay not to do that for now.